Source/Scene/JobType.js

```diff
@@ -0,0 +1,23 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    var JobType = {
+        TEXTURE : 0,
+        PROGRAM : 1,
+        BUFFER : 2,
+        NUMBER_OF_JOB_TYPES : 3
+        // Potential additional types:
```

Comment:
This smells like a TODO, even if the word isn't there.  Should we write an issue for this and remove the comment?